cmd/release_timesheet.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2017 Julien Pivotto <roidelapluie@inuits.eu>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/inuits/12to8/api"
+	"github.com/spf13/cobra"
+)
+
+// timesheetReleaseCmd represents the timesheetRelease command
+var releaseTimesheetCmd = &cobra.Command{
+	Use:   "timesheet [MM[/YYYY]]",
+	Short: "Releate a timesheet to pending state",
+	Long: `Release a timesheet for approval. Once you have done this,
```

Comment:
the short and long descriptions dont tell the same story here

Reply:
improved